docker-compose.yml

```diff
@@ -1,301 +1,37 @@
 version: "3.7"
 services:
```

Comment:
looks like this was removed for local testing?

Reply:
Fixed